opennms-webapp-rest/src/test/java/org/opennms/web/rest/v1/NodeRestServiceIT.java

```diff
@@ -180,6 +180,20 @@ public void testNode() throws Exception {
         assertTrue(xml.contains("<sysContact>OpenNMS</sysContact>"));
         assertTrue(xml.contains("<operatingSystem>MacOSX Leopard</operatingSystem>"));
 
+        // Testing individual node rescan
+        String rescanUrl = url + "/rescan";
+        m_mockEventIpcManager.getEventAnticipator().reset();
+        m_mockEventIpcManager.getEventAnticipator().anticipateEvent(new EventBuilder(EventConstants.RELOAD_IMPORT_UEI, "Test")
+                .setNodeid(1)
+                .getEvent());
+
+        xml = sendRequest(PUT, rescanUrl, 200);
+
+        m_mockEventIpcManager.getEventAnticipator().waitForAnticipated(10000);
+        m_mockEventIpcManager.getEventAnticipator().verifyAnticipated();
+
+        assertNotNull(xml); // TODO - what should the response contain / what should we check for
```

Comment:
The output is expected to be empty, right? If it does not matter, just remove the TODO.